src/scss/layouts/_catalogue.scss

```diff
@@ -74,17 +74,13 @@ section {
   .navbar-container {
```

Comment:
Our goal is to completely eliminate all scss files. Make sure to clean those up entirely. If things don't work out, please let me know!